src/static/helpers/slasHelper.ts

```diff
@@ -166,6 +166,7 @@ export async function authorize(
  * @param credentials.clientSecret - secret associated with client ID
  * @param parameters - parameters to pass in the API calls.
  * @param parameters.usid? - Unique Shopper Identifier to enable personalization.
+ * @param parameters.dnt? - Optional parameter to enable Do Not Track (DNT) for the user.
```

Comment:
Since it is boolean, does it make sense to set default value instead of making the parameter optional?`dnt=true`

Reply:
the default is controlled at the API level, if this parameter is not present, SLAS default to `dnt=false`.